File: FixDB.java
import java.sql.*;
public class FixDB {
    public static void main(String[] args) {
        String url = "jdbc:postgresql://db.ggcecwmxxuhadtihmjxf.supabase.co:5432/postgres?ssl=true&sslmode=require&sslfactory=org.postgresql.ssl.NonValidatingFactory";
        try (Connection conn = DriverManager.getConnection(url, "postgres", "Medisync2024")) {
            Statement stmt = conn.createStatement();
            int rows = stmt.executeUpdate("UPDATE doctors SET institutional = false WHERE hospital_entity_id IS NULL AND institutional = true;");
            System.out.println("Rows updated: " + rows);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: fix_db.py
import psycopg2

conn = psycopg2.connect(
    dbname="postgres",
    user="postgres",
    password="Medisync2024",
    host="db.ggcecwmxxuhadtihmjxf.supabase.co",
    port="5432"
)
cur = conn.cursor()
cur.execute("UPDATE doctors SET institutional = false WHERE hospital_id IS NULL AND institutional = true;")
print(f"Rows updated: {cur.rowcount}")
conn.commit()
cur.close()
conn.close()
